streams/src/test/java/org/apache/kafka/streams/integration/ResetPartitionTimeIntegrationTest.java

```diff
@@ -91,6 +91,7 @@ public static void closeCluster() {
     private static final int DEFAULT_TIMEOUT = 100;
     private static long lastRecordedTimestamp = -2L;
 
+    @SuppressWarnings("deprecation")
```

Comment:
nit (below) replace `_beta` with `_v2` (even if it does not avoid the need to suppress warnings...